manage.py

```diff
@@ -22,5 +39,74 @@ def run_manual(category):
     else:
         print("[-] No methods available.")
 
+@manager.command
+def map_entities():
+    """
+    Run the article crawler for all newspapers.
+    """
+    db = DBManager.get_redis()
+
+    entity_count = dm.get_entity_count()
+    for key in range(1, entity_count + 1):
+        if key % 50 == 0: print("[+] " + str(key) + "th data is processing...")
+        entity = dm.get_entity_by_key(key)
+        db.hset("entity_map", entity.entity_id, key)
```

Comment:
Hm. Shouldn't the entity table keyed by entity_id? What is this map for?

Reply:
`entity_map` is (Wikipedia entity ID -> `entity_id`) hash. We need it to build relationships between entity and article.